### DISCUSSION TITLE
Cluster Installation Error: machine config pools are degraded

### DISCUSSION THREAD

**Discussion by biruand1016** [okd-project/okd#1370] [Q&A]
Hello,
I had installed okd:4.11.0-0.okd-2020-10-15-073651 on one VM with restricted network without internet.
I created mirror-registry and mirrored images by below command
`oc adm release mirror -a /root/.docker/config.json --from=quay.io/openshift/okd:4.11.0-0.okd-2022-10-15-073651 --to=mirror.registry.com:8443/init/okd  --to-release-image=mirror.registry.com:8443/init/okd:4.11.0-0.okd-2022-10-15-073651
`
Then I  have installed cluster successfully. I confirmed this by run command
`openshift-install wait-for bootstrap-complete --dir /opt/mirror --log-level debug and openshift-install wait-for install-complete --dir /opt/mirror --log-level debug
`
But in fact, there are errors
Unable to apply 4.11.0-0.okd-2022-10-15-073651: error during syncRequiredMachineConfigPools: [timed out waiting for the condition, error pool master is not ready, retrying. Status: (pool degraded: true total: 1, ready 0, updated: 0, unavailable: 1)]
**```
**oc get nods**
master0.ocd4.rnur.edu   Ready    master,worker   16h   v1.24.6+5157800
**oc get mcp**
master                                                      False     True       True       1              0                   0                     1                      16h
worker   rendered-worker-9386356fb7336da6f317b41f9276baff   True      False      False      0              0              0             0               16h
**oc get clusteroperators**
...
**```
openshift-marketplace                              community-operators-6r2p6                                   0/1     ImagePullBackOff   0             16h
openshift-marketplace                              community-operators-97jf2                                   0/1     ImagePullBackOff   0           16h
```**
....
```**
Cluster should not be upgraded between minor versions for multiple reasons: AdminAckRequired,DegradedPool
    Kubernetes 1.25 and therefore OpenShift 4.12 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6955381 for details and instructions.
    Cluster operator machine-config should not be upgraded between minor versions: One or more machine config pools are degraded, please see oc get mcp for further details and resolve before upgrading
Cluster operator machine-config should not be upgraded between minor versions: One or more machine config pools are degraded, please see `oc get mcp` for further details and resolve before upgrading
Error while reconciling 4.11.0-0.okd-2022-10-15-073651: the cluster operator machine-config is degraded
Unable to retrieve available updates: Get "https://amd64.origin.releases.ci.openshift.org/graph?arch=amd64&channel=stable-4&id=d989ff52-68c9-435e-8b88-c35b0fc1c2cd&version=4.11.0-0.okd-2022-10-15-073651": dial tcp: lookup amd64.origin.releases.ci.openshift.org on 172.30.0.10:53: server misbehaving

why was this issue caused?, and How can I fix?

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)
